### ISSUE TITLE
Using `babel-preset-env` with `modules: false` causes issues with ES6 imports

### ISSUE THREAD

**Issue by Maruz** [storybooks/storybook#957]
Hey, I have spent some time figuring out why I could not get storybook to work, both in an existing project and a project I had just set up with some minimal configuration in place.

This is the error I encountered:
![image](https://cloud.githubusercontent.com/assets/870353/25333136/6e86b294-28e9-11e7-86f8-115cf4a68a71.png)

When searching for a solution to this issue, it looks like there are others who have had similar issues, but I did not find a solution that worked for me, so I am posting my findings here.

The difference with my babel config compared to the default one provided by storybook is that I use `babel-preset-env` rather than `babel-preset-es2015` and `babel-preset-es2016`.

Turns out the `.babelrc` file in my project root, specifically the env preset configured with `modules: false` causes issue with ES6 style imports. 

```
{
  "presets": [
    ["env", {
        "modules": false
    }],
    "react",
    "stage-0"
  ]
}
```

I have modules disabled in my config to make use of webpack 2 features for optimizing the generated bundles. 

I eventually found out that the solution is to create a `.storybook/.babelrc` file where i override the presets from my project root:

```
{
  "presets": [
    "env",
    "react",
    "stage-0"
  ]
}
```

I suggest that a section concerting `babel-preset-env` and `modules: false` configuration could be added to the Babel section of this page: https://storybooks.js.org/docs/react-storybook/configurations/default-config

With webpack 2 out, and babel encouraging people to use the env preset this issue is bound to show up for more and more people.

**Comment by ndelangen** [comment]
Compatibility with this is coming in 3.0.0

**Comment by ndelangen** [comment]
I'll close this issue, as it's been fixed in code, and will be in a 3.0.0-alpha.01 release soon!
